modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -15,6 +15,11 @@ The most common use case for this functionality is to specify a static IP addres
 * Configure a `PTR` DNS record with a valid hostname for each node with a static IP address.
 * Install the NMState CLI (`nmstate`).
 
+[IMPORTANT]
+====
+If you're using a provisioning network, it needs to be configured by Ironic's `dnsmasq`, which requires virtual media to configure a fully static deployment.
```

Comment:
The "which" between the two statements suggests that they're referring to the same thing. They're two completely separate things though - either you use a provisioning network with dnsmasq, or you are doing a fully static deployment with virtual media. I would suggest something like:

If you're using a provisioning network, it needs to be configured by Ironic's `dnsmasq`. In order to do a fully static deployment, you must use virtual media.